src/suggestion.rs

```diff
@@ -50,14 +83,216 @@ impl fmt::Display for Detector {
     }
 }
 
+// For long lines, literal will be trimmed to display in one terminal line.
+// Misspelled words that are too long shall also be ellipsized.
+pub fn condition_display_content(
+    terminal_size: usize,
+    indent: usize,
+    stripped_line: &str,
+    mistake_range: Range,
+    padding_till_excerpt_start: usize,
```

Comment:
@laysauchoa I do not understand the difference of `padding_till_excerpt_start` and `chars_till_start_statement`? Is the one `chars_till_start_statement` the number of characters within `stripped_line` until we reach the line?

Reply:
`padding_till_excerpt_start`

```
    // The paddings give some space for the ` {} ...` and extra indentation and formatting:

    //|-----| padding_till_excerpt_start
    //    |
    // 62 |  ... Reasn of food, what's up with pie? There's strawberry pie, apple, pumpkin ...

    // indent = 3 + line_number_digit_count
    //padding_till_excerpt_start = indent + 2 white spaces = 7usize, for this case.
    // 3 from spaces around the digit and the |
    // 2 from the empty spaces until we start to write the sentence
```
I have checked again and we are currently using 3 empty spaces until the sentence starts. Could it be that change from the previous display.

```
error: spellcheck(Hunspell)
  --> /home/tmhdev/Documents/cargo-spellcheck/src/documentation/markdown.rs:3
   |
 3 |   Resulting overlay is plain and can be fed into a grammer or spell checker.
   |                                                     ^^^^^^^
   | - grimmer, grammar, crammer, programmer, Grahame, or aerogram
   |
   |   Possible spelling mistake found.
   |
```

Reply:
`chars_till_start_statement`
It is an attempt to recalculate where the range of the word would starts compared to the single line itself not to the chunk, mainly to not depend on in the chunk but only in the `stripped_line`.